### DISCUSSION TITLE
Is there a way to navigation using hjkl for switcher and/or recent files?

### DISCUSSION THREAD

**Discussion by jaideeprana** [JetBrains/ideavim#371] [General]
Currently I have to use arrow keys to do so, which I personally don't prefer.

VS code already supports this feature.

Sharing screen shot of the menus that I am referring about:

![image](https://user-images.githubusercontent.com/3875065/135040169-31d7cbf5-e41b-492a-b5fc-e94ab25e9172.png)

![image](https://user-images.githubusercontent.com/3875065/135040460-61b15953-80c2-4fc1-b6c0-3148888c1327.png)

**Comment by AlexPl292** [comment]
Hi! No, unfortunately, this is not yet supported by IdeaVim.
https://youtrack.jetbrains.com/issue/VIM-2347

**Reply by jaideeprana** [reply]
Thanks.

Is there a way I can upvote for this feature request somewhere?

**Reply by AlexPl292** [reply]
Yes, using a thumb button 
![image](https://user-images.githubusercontent.com/4203721/135071100-fddc28c0-f7bd-4bdc-8720-b5cc0169cbb1.png)

**Reply by jaideeprana** [reply]
Shall I update the category as well?

![image](https://user-images.githubusercontent.com/3875065/135071782-8adf07f2-8223-4bb6-a25e-02d56020ac43.png)

From question to ideas?

**Reply by AlexPl292** [reply]
I'll move it to Z-Archive. We keep the mail issues list in YouTrack.
